north-star-vision.md

```diff
@@ -0,0 +1,137 @@
+# North Star Vision
+
+## SIG Release Roadmap for 2021 and beyond
+
+This document contains the SIG Release Roadmap for 2021. The status tracking can
+be found at the bottom. More detailed information can be found on the [SIG
+Release][0] and [Release Engineering][1] project boards.
+
+[0]: https://github.com/orgs/kubernetes/projects/23
+[1]: https://github.com/orgs/kubernetes/projects/30
+
+### Primary Focus
+
+Establish a **consumable**, **introspectable**, and **secure** supply chain for
+Kubernetes. As a supply chain we understand the defining, building and
+publishing of Kubernetes related artifacts.
+
+1. **Consumable**: Improving the usability of artifacts by making their
+   consumption easier. This includes being process independent from external
+   companies like Google.
+1. **Introspectable**: It is clear for users at which point and how Kubernetes
+   artifacts are being built. This includes the documentation of all
```

Comment:
Are reproducible build artifacts included as part of this? I don't see them mentioned below. That has been a user-facing issue in the past (e.g. not being able to restart/recover from an issue during the release process and having to cut additional tags)

Reply:
Right now it's not part of the roadmap. I see that https://github.com/kubernetes/kubernetes/issues/70131 has been closed, but there is an open question at the end of the conversation. Maybe we want to add this topic as well. Thoughts on that @kubernetes/release-team-leads ?